### PR TITLE
[4.0] template description

### PR DESCRIPTION
**Pull request by brianteeman**
correct an error and sync the language files

### CODE REVIEW

File: language/en-GB/en-GB.tpl_cassiopeia.ini [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-TPL_CASSIOPEIA_XML_DESCRIPTION="Continuing the space theme (Solarflare from 1.0 and Milkyway from 1.5), Cassiopeia is the Joomla 4 site template based on Bootstrap 4."
+TPL_CASSIOPEIA_XML_DESCRIPTION="Continuing the space theme (Solarflare from 1.0, Milkyway from 1.5 and Protostar for 3.0), Cassiopeia is the Joomla 4 site template based on Bootstrap 4."
```
Comment (Quy) [inline language/en-GB/en-GB.tpl_cassiopeia.ini]: `for 3.0` to `from 3.0`?

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 969dceb9039a669cc66d983428a398a4dece21ff<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18631">issues.joomla.org/tracker/joomla-cms/18631</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**